issuance/issuance.go

```diff
@@ -626,3 +626,45 @@ func RequestFromPrecert(precert *x509.Certificate, scts []ct.SignedCertificateTi
 		SCTList:           scts,
 	}, nil
 }
+
+// LoadChain takes a list of filenames containing pem-formatted certificates,
+// and returns a chain representing all of those certificates in order. It
+// ensures that the resulting chain is valid. The final file is expected to be
+// a root certificate, which the chain will be verified against, but which will
+// not be included in the resulting chain.
+func LoadChain(certFiles []string) ([]*x509.Certificate, error) {
```

Comment:
Since this is in the issuance package, and since we know that every single cert in the chain is an issuer certificate, return a slice of issuance.Certificates instead of a slice of x509.Certificates
```suggestion
func LoadChain(certFiles []string) ([]*Certificate, error) {
```

Reply:
omg... how could I have missed this!